core/tracing/hooks.go

```diff
@@ -107,6 +107,9 @@ type (
 	// BlockchainInitHook is called when the blockchain is initialized.
 	BlockchainInitHook = func(chainConfig *params.ChainConfig)
 
+	// CloseHook is called when the blockchain closes.
+	CloseHook = func()
```

Comment:
Do we want to allow returning an error? In general Go's Close() methods do so

Reply:
None of the other tracing methods return an error or any value for that matter. My 2c keep it as is.